### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by olliethomas**

### CODE REVIEW

File: numpy-stubs/__init__.pyi [hunk 1]
```diff
@@ -773,10 +773,10 @@ class matrix:
...
 # Specific values
 #
-inf: float
-nan: float
-NaN: float
-NAN: float
+inf: float32
+nan: float32
+NaN: float32
+NAN: float32
```
Comment (thomkeh) [inline numpy-stubs/__init__.pyi:779]: this doesn't seem true?

```python
type(np.inf) == float
```

File: pandas-stubs/core/frame.pyi [hunk 2]
```diff
@@ -184,7 +198,7 @@ class DataFrame:
     def rename(self, mapper: Callable, axis: _AxisType = ...) -> DataFrame: ...
     @overload
     def rename(self, columns: Dict[_str, _str]) -> DataFrame: ...
-    def replace(self, a: float, b: float) -> DataFrame: ...
+    def replace(self, a: _DTypeEx, b: _DType, regex: bool = ...) -> DataFrame: ...
```
Comment (thomkeh) [inline pandas-stubs/core/frame.pyi:201]: Why use a TypeVar here? Just do
```suggestion
    def replace(self, a: Union[_np.dtype, str], b: Union[_np.dtype, str], regex: bool = ...) -> DataFrame: ...
```

TypeVars are only needed if the type of the arguments influence the return type

File: numpy-stubs/__init__.pyi (hunk 1 above)
Comment (olliethomas) [reply numpy-stubs/__init__.pyi]: yeah....I was trying to get around a problem by introducing a new problem/error. Resolved

File: pandas-stubs/core/frame.pyi (hunk 2 above)
Comment (olliethomas) [reply pandas-stubs/core/frame.pyi]: good point

**Pull request merged**